cmd/contact-exporter/main.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"gopkg.in/gorp.v1"
+
+	"github.com/jmhodges/clock"
+	"github.com/letsencrypt/boulder/cmd"
+	blog "github.com/letsencrypt/boulder/log"
+	"github.com/letsencrypt/boulder/sa"
+)
+
+type contactExporter struct {
+	log   blog.Logger
+	dbMap *gorp.DbMap
+	clk   clock.Clock
+}
+
+func (c contactExporter) findContacts() ([]string, error) {
+	var contactsJSON []string
```

Comment:
I think it would need to be a `[][]byte` so that the `range` iterates over `[]byte` to give to `json.Unmarshal`, will try taht.